section3/reflection.md

```diff
@@ -1,17 +1,33 @@
 ## Section 3 Reflection
 
-1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
+1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique? <br>
+Felt a little bit different, I didn't do much during the breaks.  Also got distracted easily
 
-2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
+2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)? <br>
+When the breaks came, I found myself doing the things I usually do when I get distracted (look at twitter, check texts, etc.)
 
-3. What is a Hash, and how is it different from an Array?
+3. What is a Hash, and how is it different from an Array? <br>
+Hash does not have order, it uses keys to return values, rather than positions.  Can store a lot of things in a hash.  Also called a lookup table
 
 4. In the space below, create a Hash stored to a variable named `pet_store`.  This hash should hold an inventory of items and the number of that item that you might find at a pet store.
+```
+pet_store = {dogs: 2, cats: 4, toys: 45}
+```
 
 5. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
+```
+states{IA}
+```
 
 6. With the same hash above, how would we get all the keys?  How about all the values?
+```
+states.keys
+states.values
+```
 
-7. What is another example of when we might use a hash?  In your example, why is a hash better than an array?
+7. What is another example of when we might use a hash?  In your example, why is a hash better than an array? <br>
+If we wanted to keep track of a vacation (length, cost, where we are going, what we are eating), <br>
+Hash is better than an array because it allows us to add more complexity
 
-8. What questions do you still have about hashes?
\ No newline at end of file
+8. What questions do you still have about hashes? <br>
+No pressing questions at the moment, but it would be nice to get some more practice like the ex39.rb file had.  Also perhaps when to use ":" vs "=>"
```

Comment:
YES in RE to when to use : vs => - that will be defined by a style guide - while in Turing you'll be provided with those expectations and on the job, it will be the same. Those style guides may vary company by company but there are usually strong opinions and developers are generally expected to get on board with those style expectations in their code.